packages/opentelemetry-context-asynchooks/tsconfig-release.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file is not needed for now. Refer to https://github.com/open-telemetry/opentelemetry-js/pull/53